x-pack/legacy/plugins/uptime/public/components/functional/charts/snapshot_histogram.tsx

```diff
@@ -124,7 +125,15 @@ export const SnapshotHistogramComponent = ({
             />
           </h5>
         </EuiTitle>
-        <ChartWrapper height={height} loading={loading}>
+        <ChartWrapper
+          height={height}
+          loading={loading}
+          aria-label={i18n.translate('xpack.uptime.snapshotHistogram.description', {
+            defaultMessage: `Bar Chart showing uptime status over time from 
```

Comment:
When injecting values into translated strings we're required to use the `values` property; this helps the translators preserve context.

```suggestion
          aria-label={i18n.translate('xpack.uptime.snapshotHistogram.description', {
            defaultMessage:
              'Bar Chart showing uptime status over time from {startTime} to {endTime}.',
            values: {
              startTime: moment(new Date(absoluteStartDate).valueOf()).fromNow(),
              endTime: moment(new Date(absoluteEndDate).valueOf()).fromNow(),
            },
          })}
```